safe/definitions/reports/components.py

```diff
@@ -346,6 +348,49 @@
     }
 }
 
+analysis_provenance_details_component = {
+    'key': 'analysis-provenance-details',
+    'type': jinja2_component_type,
+    'processor': jinja2_renderer,
+    'extractor': analysis_provenance_details_extractor,
+    'output_format': Jinja2ComponentsMetadata.OutputFormat.String,
+    'output_path': 'analysis-provenance-details-output.html',
+    'template': 'standard-template/'
+                'jinja2/'
+                'analysis-provenance-details.html',
+    'extra_args': {
+        'defaults': {
+            'source': tr('source not available'),
+            'reference': tr('reference unspecified'),
+            'aggregation_not_used': tr('not used')
+        },
+        'header': {
+            'analysis_detail': tr('Analysis details')
+        },
+        'provenance_format': {
+            'hazard_header': tr(
+                'Hazard source'),
+            'hazard_format': tr(
+                '{layer_name} - {source} - '),
+
+            'exposure_header': tr(
+                'Exposure source'),
+            'exposure_format': tr(
+                '{layer_name} - {source} - '),
+
+            'aggregation_header': tr(
+                'Aggregation source'),
+            'aggregation_format': tr(
+                '{layer_name} - {source} - '),
+
+            'impact_function_header': tr(
+                'Impact Function'),
+            'impact_function_format': tr(
+                '{impact_function_name}'),
```

Comment:
In transifex we will copy paste the block because there is nothing to translate here.
So to have this tr to works, we would need to put every combinations of hazard/exposure in https://github.com/inasafe/inasafe/blob/develop/safe/common/dynamic_translations.py
This would be a waste of time ;-)

 I just put a commit to translate the name of the IF before your side : https://github.com/inasafe/inasafe/pull/4041/commits/1670822f2ac3de1f5bc9db28d773e1e9865977cc 

So i think it's safe if you try to translate this ;-)